maven-compat/pom.xml

```diff
@@ -80,6 +80,10 @@ under the License.
       <groupId>javax.inject</groupId>
       <artifactId>javax.inject</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.apache.maven</groupId>
+      <artifactId>plexus-utils</artifactId>
+    </dependency>
```

Comment:
This was removed in 703f814e34a04924ecec923d8f45508600d97b0f, [restored as per this thread](https://github.com/apache/maven/pull/1066#discussion_r1141462379), and similar for maven-embedder below. Not a net-new dependency, but explicit direct dependency that also comes in transitively.